cves/CVE-2011-3192.yml

```diff
@@ -140,10 +149,14 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: |
+    The most significant event is that which occurred when the vulnerability was introduced.
+    It would seem that at this point in time, the httpd was undergoing a major set of refactors
+    to reduce the size of their codebase and to make things easier to understand. During this refactor,
+    it would seem that logic for handling requests was adjusted and the vulnerability introduced.
```

Comment:
**Grammar**
...and the vulnerability `was` introduced.

Reply:
I don't know how I missed that! Addressed.